### DISCUSSION TITLE
train_val 실험결과 공유(Cascade r-cnn)

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#75] [Show and tell]
결론: valid와 submission 차이가 크지 않은 좋은 valid set인 것 같습니다.

실험결과: https://wandb.ai/cider6/detr/runs/159tbiob?workspace=user-torque
데이터셋 경로: /opt/ml/detection/mmconfig/sample/cascade_rcnn_trainval_test/
브랜치: train_val_test
EDA상 큰 문제가 없어 학습 수행

제출 성능: 0.453
bbox_map_75: 0.37 -> delta 0.083
bbox_map_50: 0.456 -> delta 0.003

감사합니다

**Comment by jaehwan-AI** [comment]
오오 차이가 굉장히 적네요
해당 데이터셋으로 retinanet도 돌려보고 결과를 비교해보는 것도 좋을 것 같아요
제가 돌려보겠습니다!

**Comment by jaehwan-AI** [comment]
해당 데이터셋으로 retinanet을 돌려본 결과입니다.

제출 성능: 0.471
bbox_map_50: 0.522 -> delta 0.051

cascade r-cnn보다 차이가 조금 나네요..
다른 데이터셋들도 돌려보면서 이 차이가 유지가 되는 지도 궁금해지네요
